v03_pipeline/lib/misc/pedigree.py

```diff
@@ -64,7 +64,7 @@ def __hash__(self):
         return hash(self.family_guid)
 
     @staticmethod
-    def parse_direct_lineage(rows: list[hl.Struct]) -> dict[str, Sample]:
+    def parse_direct_lineage(rows: list[hl.Struct]) -> dict[str, Sample]:  # noqa: C901
```

Comment:
For context, I'm running off of this code to parse the pedigrees, but the actual sex/relatedness checking is [disabled via env var](https://github.com/broadinstitute/seqr-loading-pipelines/blob/main/v03_pipeline/lib/tasks/write_remapped_and_subsetted_callset.py#L111-L114)